src/demo/assetPath.js

```diff
@@ -0,0 +1,5 @@
+global.__ml_activities_asset_public_path__ = './assets/';
+
+export const setAssetPath = (path) => {
+  global.__ml_activities_asset_public_path__  = path;
```

Comment:
This is the magic that allows us to use a different asset prefix when running under Code Studio. This API will be called by Code Studio before it does a runtime `require('@code-dot-org/ml-activities')`